addons/ofxKinect/libs/libfreenect/src/loader.c

```diff
@@ -169,7 +169,7 @@ FN_INTERNAL int upload_firmware(fnusb_dev* dev, char * filename) {
 				if (!home)
 					continue;
 				int homelen = strlen(home);
-				char* dotfolder = "/.libfreenect";
+				char dotfolder[14] = "/.libfreenect";
```

Comment:
fixes C17 issues  with this addon - as char* became const in new standard. this fixes it